code/index.html

```diff
@@ -2,21 +2,129 @@
+        </p>
+        <a href="#">Read more ></a>
+
+      </div>
+    
+
+      <div class="other-news-card">
+        <img class="other-news-image" src="images/sadcat.jpg" alt="sad cat"/>
+          <h3>Sad cats make Nasdaq crash</h3> 
+          <p class="other-news-text">
+            During the pandemic we have all been a little bit gloomy, but the special japanese sad-cat have even had an influence of the global stock market.
+          </p>
+          <a href="#">Read more ></a>
+      </div>
+
+
+      <div class="other-news-card">
+        <img class="other-news-image" src="images/youngfolks.jpg" alt="five young persons walking"/>
+          <h3>Young folks immune to ads</h3> 
+          <p class="other-news-text">
+            One would think that the generations growing up today would be wildly influenced by influencers? But no – it is quite the opposite.
+          </p>
+          <a href="#">Read more ></a>
+      </div>
+
+  
+      <div class="other-news-card">
+        <img class="other-news-image" src="images/brit.jpg" alt="Britney Spears smiling"/>
+          <h3>Britney Spears is on a roll</h3> 
+          <p class="other-news-text">
+            After intense years things are finally looking brighter for Brit! The last year she has ended the long-lasting conservatorship, regain custody of her kids - and is now pregnant!
+          </p>
+          <a href="#">Read more ></a>
+      </div>
+
+      <div class="other-news-card">
+        <img class="other-news-image" src="images/happybday.jpg" alt="candles in a birthdaycake"/>
+          <h3>The world's oldest woman is older</h3> 
+          <p class="other-news-text">
+            Here’s something worth celebrating! The oldest woman in the world, Myra Myrdotter, turns 122 today. Happy happy birthday! We know how to celebrate properly!
+          </p>
+          <a href="#">Read more ></a>
+      </div>
+
+      <div class="other-news-card">
+        <img class="other-news-image" src="images/ducklings.jpg" alt="duck and ducklings in a row"/>
+          <h3>Ducklings thriving in roundabout</h3> 
+          <p class="other-news-text">
+            It is not the safest of homes, but these ducklings are growing and thriving despite their somewhat unconventional nest. Since May, they have all been living in a roundabout.
+          </p>
+          <a href="#">Read more ></a>
+      </div>
+
+      <div class="other-news-card">
+        <img class="other-news-image" src="images/hedgie.jpg" alt="cute little hedgehog"/>
+          <h3>Hedgehogs instead of medicine</h3> 
+          <p class="other-news-text">
+            Well, ok. That's a misleading headline. But two hedgehogs made life in quarantine a lot more fun and cute for the elderly in Melbourne.
+          </p>
+          <a href="#">Read more ></a>
+      </div>
+
+       
     </section>
-  </body>
+    
+</body>
```

Comment:
Very nice job with all the divs, classes and names. Everything follows the guidelines. Amazing job!!